src/styles/app.css

```diff
@@ -0,0 +1,10 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+
+.nasa-logo {
+  margin: auto;
+  width: 500px;
+  height: 300px;
```

Comment:
You dont actually need both of these properties here, you can just use one and the image will scale appropriately. I tend to use width given the choice